etc/mosquitto_udmi.conf

```diff
@@ -11,3 +11,6 @@ cafile /etc/mosquitto/certs/ca.crt
 certfile /etc/mosquitto/certs/rsa_private.crt
 keyfile /etc/mosquitto/certs/rsa_private.pem
 require_certificate true
+
+listener 1883
```

Comment:
what's this for?  Were you actively using it somewhere or is it a debugging hook?

Reply:
Yes this is for local development with containers until we implement support for CA certificates.

Reply:
Port seems to be occupied by CI, reverted this change.